src/utils/hash.ts

```diff
@@ -40,6 +41,16 @@ export async function hashValues(payload: any, jwt: string) {
   return result;
 }
 
+export async function hashValue(value: string) {
```

Comment:
can we reuse this function inside the hashAndEncryptValues function?

Reply:
Definitely. But we'll be doing that nasty sodium ready check in the loop.

Reply:
right.   I did see that.